buildtest/cli/__init__.py

```diff
@@ -149,7 +149,22 @@ def valid_time(value):
     return dt_object
 
 
-def get_parser():
+class ParentParser:
```

Comment:
I don't see a point in having a class if all it does it holds a single variable. Can't we just declare this dictionary anywhere even a method would do which just returns the parent parser object.

The main thing is we need the parent parser object to be accessible in get_parser and any of the sub commands where we build the command menu